var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -41,6 +41,9 @@ class Ncurses(AutotoolsPackage):
     variant('symlinks', default=False,
             description='Enables symlinks. Needed on AFS filesystem.')
 
+    variant('wide', default=True,
```

Comment:
This should be a multi-valued variant (as in #3852), not a boolean variant.

A boolean variant is interpreted by the concretizer as ADDING something to the package.  In this case, setting the boolean variant to True SUBTRACTS something from the package.  This leads to using a tilde in `depends_on()` declarations, which can make things not concretize when they otherwise should.